src/js/components/Select/doc.js

```diff
@@ -0,0 +1,60 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { a11yTitlePropType, backgroundPropType } from '../utils';
+
+export default (Select) => {
+  const DocumentedSelect = describe(Select)
+    .description(
+      'An select-like field with optional search capability.'
+    ).usage(
+      `import { Select } from 'grommet';
+      <Select/>`
+    );
+
+  DocumentedSelect.propTypes = {
+    a11yTitle: a11yTitlePropType,
+    background: backgroundPropType,
+    basis: PropTypes.string.description(
```

Comment:
`basis` seems like an odd name for this property. I understand how it is being used, but the caller isn't aware of the context. Can we not automatically handle this without having the caller specify it? Could we could do something like `flex: 0 1 80vh` all the time?

Reply:
let me try and see if it works. great suggestion!